UI/org.eclipse.birt.report.designer.core/src/org/eclipse/birt/report/designer/core/model/schematic/ImageHandleAdapter.java

```diff
@@ -145,7 +159,21 @@ public Dimension getRawSize() {
 			}
 		}
 
-		return new Dimension(Math.max(px, 0), Math.max(py, 0));
+		// proportional scale of the image size
+		if (this.getImageHandle().isProportionalScale()) {
+			py = px;
```

Comment:
I don't understand the code, but my gut feeling is that this might be wrong, even though my test report looks right.

Can you please also test reports where the image itself is not a square?